---
title: Blok "Catch" nigdy nie został osiągnięty, ponieważ element " <name1> " dziedziczy po elemencie " <name2> "
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 06f3b3cb69d2df5d123879fa0df3dffcdc661678
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: pl-PL
ms.lasthandoff: 09/23/2020
ms.locfileid: "91084583"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a>Blok "Catch" nigdy nie został osiągnięty, ponieważ element " \<name1> " dziedziczy po elemencie " \<name2> "

`Catch`Blok w kodzie nigdy nie został osiągnięty.  
  
 Domyślnie ten komunikat jest ostrzeżeniem. Aby uzyskać więcej informacji na temat ukrywania ostrzeżeń lub leczenia ostrzeżeń jako błędów, zobacz [Konfigurowanie ostrzeżeń w Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).  
  
 **Identyfikator błędu:** BC42029  
  
## <a name="to-correct-this-error"></a>Aby poprawić ten błąd  
  
- Usuń blok nadmiarowy `Catch` .  
  
## <a name="see-also"></a>Zobacz także

- [Jak przechwytywać wyjątki przy użyciu bloku try/catch](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [Try...Catch...Finally, instrukcja](../language-reference/statements/try-catch-finally-statement.md)
